### ISSUE TITLE
solver bug?

### ISSUE THREAD

**Issue by misaka23** [modelscope/ClearerVoice-Studio#165]
在ClearerVoice-Studio/train/speech_enhancement/solver.py中

            # Check whether to early stop and to reduce learning rate
            find_best_model = False
            if val_loss >= self.best_val_loss:
                self.best_val_loss = val_loss
                self.val_no_impv += 1
                if self.val_no_impv == 5:
                    self.halving = True
                elif self.val_no_impv >= 10:
                    if self.print: print("No imporvement for 10 epochs, early stopping.")
                    break
            else:
                self.val_no_impv = 0
                self.best_val_loss = val_loss
                find_best_model=True

  if val_loss >= self.best_val_loss:下面更新 self.best_val_loss会导致lr一直不降低吧
